Example/Tests/TestHelpers/MockModel.swift

```diff
@@ -1,18 +1,32 @@
 import Faro
 
 class MockModel: Parseable {
-    var value: String
-    
+    var uuid: String
+
     required init?(from raw: Any) {
         guard let json = raw as? [String: String] else {
             return nil
         }
-        
-        value = json["key"]!
+
+        do {
+            uuid = try parseString("uuid", from: json)
+        } catch {
+            return nil
+        }
     }
     
     var JSON: [String: Any]? {
         return nil
     }
 
+    class func extractRootNode(from json: Any) -> JsonNode {
```

Comment:
❓I don't get it...

Seems to complicated to me if you have to do this in every model... Maybe we need to look [here](https://github.com/devlucky/Kakapo) to get it fixed.